src/ObjCRuntime/Runtime.cs

```diff
@@ -1594,6 +1594,21 @@ internal static MethodInfo FindClosedMethod (Type closed_type, MethodBase open_m
 
 			throw ErrorHelper.CreateError (8003, "Failed to find the closed generic method '{0}' on the type '{1}'.", open_method.Name, closed_type.FullName);
 		}
+
+		[EditorBrowsable (EditorBrowsableState.Never)]
```

Comment:
Shouldn't this be in the `#if` block?

Reply:
No, it applies to either the P/Invoke or the manual implementation.